### PR TITLE
added my file in Submissions

### PR DESCRIPTION
**Pull request by RaduStana**

### CODE REVIEW

File: Submissions/StanaR.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Stana Radu",
+  "img": "[YOUR_IMG_URL]",
```
Comment (sophiabrandt) [inline Submissions/StanaR.json:3]: Can you please remove the placeholders? You can just add an empty string:
```json
"img":  "",
```

File: Submissions/StanaR.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "email": "stana.radu@gmail.com",
+  "links": {
+    "website": "[YOUR_WEBSITE_URL]",
```
Comment (sophiabrandt) [inline Submissions/StanaR.json:6]: Same here.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

**Comment by sophiabrandt** [comment]
@RaduStana are you still interested in this PR? Otherwise, I'll close it soon.

**Comment by RaduStana** [comment]
Hello,
Yes, I am interested.
I have removed the placeholders.

În sâm., 2 mar. 2024 la 14:37, Sophia Brandt ***@***.***> a
scris:

**Pull request merged**